### DISCUSSION TITLE
Supabase Relationships Documentation

### DISCUSSION THREAD

**Discussion by paulochang** [supabase/supabase#4512] [Feature Requests]
I couldn't find any piece of documentation  showing the foreign key button on the CREATE TABLE screen (see image)
<img width="305" alt="image" src="https://user-images.githubusercontent.com/4237241/146388808-9e5658db-e9cc-4c89-8794-12097476ab05.png">
<img width="454" alt="image" src="https://user-images.githubusercontent.com/4237241/146390234-d9457800-e027-4592-8a29-042de5b1d3fa.png">
It took me some time to realize how to create relationships from the CREATE TABLE  screen. 
It would be much better, IMHO if you could add a reference to this small button in the https://supabase.com/docs/guides/database#relationships guide.

**Comment by kk21** [comment]
I find switching to Light theme makes it easier to view the supabase website and dashboard.
